src/protagonist/API/Features/DeliveryChannels/Requests/CreateCustomerDefaultDeliveryChannel.cs

```diff
@@ -0,0 +1,97 @@
+﻿using API.Infrastructure.Requests;
+using DLCS.Core;
+using DLCS.HydraModel;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.DeliveryChannels.Requests;
+
+public class CreateCustomerDefaultDeliveryChannel : IRequest<ModifyEntityResult<CreateDefaultDeliveryChannelResult>>
+{
+    public int Customer { get; }
+    
+    public int Space { get; }
+    
+    public DefaultDeliveryChannel DefaultDeliveryChannel { get; }
+    
+    public CreateCustomerDefaultDeliveryChannel(int customer, int space, DefaultDeliveryChannel defaultDeliveryChannel)
+    {
+        Customer = customer;
+        DefaultDeliveryChannel = defaultDeliveryChannel;
+        Space = space;
+    }
+}
+
+public class CreateDefaultDeliveryChannelResult
+{
+    public DLCS.Model.DeliveryChannels.DefaultDeliveryChannel? DefaultDeliveryChannel;
+}
+
+public class CreateCustomerDefaultDeliveryChannelHandler : IRequestHandler<CreateCustomerDefaultDeliveryChannel,
+    ModifyEntityResult<CreateDefaultDeliveryChannelResult>>
+{
+    private readonly DlcsContext dbContext;
+
+    public CreateCustomerDefaultDeliveryChannelHandler(DlcsContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+
+    public async Task<ModifyEntityResult<CreateDefaultDeliveryChannelResult>> Handle(
+        CreateCustomerDefaultDeliveryChannel request, CancellationToken cancellationToken)
+    {
+        var existingPolicy = await dbContext.DefaultDeliveryChannels.AnyAsync(p => 
+                p.Customer == request.Customer &&
+                p.MediaType == request.DefaultDeliveryChannel.MediaType &&
+                p.Space == request.Space,
+            cancellationToken);
+
+        if (existingPolicy)
```

Comment:
Is this initial lookup required, or would the unique constraint throw an exception?

Reply:
It throws an exception, but returns a `500` as opposed to a `409`